scapy/plist.py

```diff
@@ -402,9 +430,9 @@ def afterglow(self, src=None, event=None, dst=None, **kargs):
             event = lambda x: x['IP'].dport
         if dst is None:
             dst = lambda x: x['IP'].dst
-        sl = {}
-        el = {}
-        dl = {}
+        sl = {}  # type: Dict[Any, Tuple[int, Any]]
```

Comment:
sl, `Dict[IPField, Tuple[int, List[ShortEnumField]]]`, dl as `Dict[IPField, ShortEnumField]`, and el as `Dict[ShortEnumField, Tuple[int, List[IPField]]]`.

Reply:
Never could've figured out those data types by myself.. Thanks! :)

Reply:
I had a look at the code and try to infer the types.